i18n/en-US.properties

```diff
@@ -10,6 +10,8 @@ ba.annotationsPost = Post
 ba.annotationsSave = Save
 # Button label for cancelling the creation of a description, comment, or reply
 ba.popups.cancel = Cancel
+# Aria label description for reply popup
+ba.popups.comment = Comment
```

Comment:
Minor, but the naming convention is `ba.popup.reply.comment` since the string is specific to the PopupReply component.